components/Questionnaire/One/index.vue

```diff
@@ -258,13 +258,24 @@
               <p class="mb-3">
                 Tuliskan daftar penyedia layanan telekomunikasi yang ada di sekitar anda (Telkomsel/XL/Tri/dan lain-lain)
               </p>
-              <textarea
-                v-model="fasilitas_desa.jaringan_telepon.operator"
-                class="form-text-area"
-                name="Daftar layanan telekomunikasi"
-                placeholder="Masukkan daftar disini"
-                rows="5"
-              />
+              <div
+                :class="{
+                  'form-text-area' : true,
+                  'form-text-area--error' : showErrorMsg
+                }"
+              >
+                <textarea
+                  v-model="fasilitas_desa.jaringan_telepon.operator"
+                  class="form-text-area--text"
+                  name="Daftar layanan telekomunikasi"
+                  placeholder="Masukkan daftar disini"
+                  :error="!!(errors.telephone)"
```

Comment:
I'm confused about where the `error` attribute comes from? 🤧

Reply:
😆 I think, I forgot to remove `error` attr. Because, before I push this, I try to make `text-area`  reusable component. 
Thanks ⭐.